pages/docs/gettingstarted.md

```diff
@@ -28,39 +28,39 @@ Currently the CLI is a node.js based command line tool which requires that our s
 * [rsync](https://download.samba.org/pub/rsync/rsync.html)
 * [ssh](http://www.openssh.com/)
 
-Once we have those setup we can install `resin` CLI using npm:
+Once we have those setup we can install `balena` CLI using npm:
 ``` bash
-$ npm install --global --production --unsafe-perm resin-cli
+$ npm install --global --production --unsafe-perm balena-cli
 ```
 
 __Note:__ Depending on you node.js installation, you may need to use administrative privileges to install the CLI.
 
 ## Configure the Image
-To allow resinOS images to be easily configurable before boot, some key config files are added to boot partition. In this step we will use the CLI to configure the network, set our hostname to `resin` and disable persistent logging, because we don’t want to kill our poor flash storage with excessive writes.
+To allow balenaOS images to be easily configurable before boot, some key config files are added to boot partition. In this step we will use the CLI to configure the network, set our hostname to `balena` and disable persistent logging, because we don’t want to kill our poor flash storage with excessive writes.
 ``` bash
-$ sudo resin local configure ~/Downloads/resin.img
+$ sudo balena local configure ~/Downloads/balena.img
 ? Network SSID super_wifi
 ? Network Key super secure password
 ? Do you want to set advanced settings? Yes
-? Device Hostname resin
+? Device Hostname balena
 ? Do you want to enable persistent logging? no
 Done!
 ```
 
-If you are not using the resin CLI, you will need to mount the boot partition of the image and edit the configuration manually.
+If you are not using the balena CLI, you will need to mount the boot partition of the image and edit the configuration manually.
 
 Edit `/boot/config.json` so it looks like this:
 ``` json
 {
   "persistentLogging": false,
-  "hostname": "resin",
+  "hostname": "balena",
 }
 ```
 
-And edit the `ssid` and `psk` values in the `/boot/system-connections/resin-sample` connection file.
+And edit the `ssid` and `psk` values in the `/boot/system-connections/balena-sample` connection file.
```

Comment:
this will still stay as `resin-sample` for now

Reply:
Confirm.